DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -968,6 +968,10 @@ public class DcMsg {
     public var messageWidth: CGFloat {
         return CGFloat(dc_msg_get_width(messagePointer))
     }
+    
+    public var duration: Double {
```

Comment:
why not use the same type as the underlying core function here? so an Integer?

Reply:
I changed it to Int